qiskit/aqua/operators/list_ops/list_op.py

```diff
@@ -170,9 +170,9 @@ def adjoint(self) -> OperatorBase:
         #  always come in pairs, so an AdjointOp holding a reference could save copying.
         if self.__class__ == ListOp:
             return ListOp([op.adjoint() for op in self.oplist],  # type: ignore
-                          **self._state(coeff=np.conj(self.coeff)))  # coeff is conjugated
+                          **self._state(coeff=self.coeff.conjugate()))  # coeff is conjugated
```

Comment:
Is there an advantage or difference of using `conjugate` over `np.conj`?

Reply:
np.conj change the type to np.int64, np.float64, or np.complex128. This type conversion is necessary.

Reply:
Here, the type should be a generic numeric type in python.

Reply:
Ah, that makes sense. Thanks for the explanation 👍🏼